mead.config.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
This file doesn't seem to be needed for #2202

Reply:
Oh, That was my fault.